examples/common/pigweed/rpc_services/Device.h

```diff
@@ -270,14 +270,46 @@ class Device : public pw_rpc::nanopb::Device::Service<Device>
 
     virtual pw::Status SetPairingState(const chip_rpc_PairingState & request, pw_protobuf_Empty & response)
     {
+        chip::ChipError err;
         if (request.pairing_enabled)
         {
-            DeviceLayer::ConnectivityMgr().SetBLEAdvertisingEnabled(true);
-            DeviceLayer::ConnectivityMgr().SetBLEAdvertisingMode(DeviceLayer::ConnectivityMgr().kFastAdvertising);
+            if (!DeviceLayer::ConnectivityMgr().IsBLEAdvertisingEnabled())
+            {
+                err = DeviceLayer::ConnectivityMgr().SetBLEAdvertisingEnabled(true);
```

Comment:
The comissioning window manager will do this already when the commissioning window is opened.  Why do we need to do it here?

Reply:
done